spec/models/journal_entry_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 RSpec.describe JournalEntry, type: :model do
+  let!(:user) { FactoryBot.create(:user) }
```

Comment:
Please study the difference between let and let!
```suggestion
  let(:user) { FactoryBot.create(:user) }
```